docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - redis
       - db
     environment:
+      # Hint: Make sure you serve all requests for the `/static/` and `/media/` paths when debug is False. See [installation](https://docs.pretalx.org/administrator/installation/#step-7-ssl) for more information
```

Comment:
I'm not convinced that adding this informaitn here is valuable – you might as well add information on every other environment variable and/or setting and/or component.